images/fluentd-elasticsearch/dockerfile-location.txt

```
@@ -0,0 +1 @@
+The Dockerfile is here: https://github.com/elastisys/fluentd-elasticsearch/
```

```suggestion
The Dockerfile is here: https://github.com/elastisys/fluentd-elasticsearch/
```

Make this a README.md so it is the thing that pops up when you enter the directory on Github.